### ISSUE TITLE
Multistage gap fit Turbo_SOAP

### ISSUE THREAD

**Issue by Felixrccs** [libAtoms/workflow#219]
I am starting to use the multistage gap fit with the turbo-soap descriptor. bapatist and I adjusted the **multistage_gap_params.json** file that you provide in the examples/iterative_gap_fit folder to be suited for the turbo soap-descriptor (in combination with the universal SOAP package). Since the turbo soap descriptor is currently quite popular with the GAP users, I think it would be wise to add our **turbosoap_multistage.json** file either to the examples or in the Documentation. As soon as I get your feedback I can do the regarding pull request.
`
{
  "stages": [
    {
      "error_scale_factor": 10.0,
      "descriptors": [
        {
          "descriptor": {
            "distance_Nb": true,
            "order": 2,
            "cutoff": "_EVAL_ {BOND_LEN_Z1_Z2}*1.5",
            "cutoff_transition_width": "_EVAL_ {BOND_LEN_Z1_Z2}*1.5/5.0",
            "compact_clusters": true,
            "Z": ["_EVAL_ {Z1}", "_EVAL_ {Z2}"]
            },
          "fit":  {
            "n_sparse": 20,
            "covariance_type": "ard_se",
            "theta_uniform": "_EVAL_ {BOND_LEN_Z1_Z2}*1.5/5.0",
            "sparse_method": "uniform",
            "f0": 0.0
            },
          "count_cutoff": "_EVAL_ {BOND_LEN_Z1_Z2}*1.4",
          "add_species": "manual_Z_pair"
        }
        ]
      },
      {
        "error_scale_factor": 1.0,
        "descriptors": [
          {
            "descriptor": {
              "soap_turbo": true,
              "alpha_max": "_EVAL_ {nZ}*[8]",
              "l_max": 4,
              "atom_sigma_r": "_EVAL_ {nZ}*[{ATOM_SIGMA}]",
	          "atom_sigma_t": "_EVAL_ {nZ}*[{ATOM_SIGMA}]",
	          "atom_sigma_r_scaling": "_EVAL_ {nZ}*[0.0]",
              "atom_sigma_t_scaling": "_EVAL_ {nZ}*[0.0]",
	          "amplitude_scaling": "_EVAL_ {nZ}*[1.0]",
              "rcut_hard": "_EVAL_ {R_CUT}",
              "rcut_soft": "_EVAL_ {R_CUT} - {R_TRANS}",
              "central_weight": "_EVAL_ {nZ}*[1.0]",
              "n_species": "_EVAL_ {nZ}",
              "species_Z": "_EVAL_ {Zs}",
              "radial_enhancement": 1,
              "basis": "poly3gauss",
              "Z": "_EVAL_ {Zcenter}"
            },
            "add_species": "manual_universal_SOAP",
            "fit": {
              "n_sparse": 1000,
              "f0": 0.0,
              "covariance_type": "dot_product",
              "zeta": 4.0,
              "sparse_method": "cur_points",
              "print_sparse_index": true
              }
          }
          ]
        }
  ],
  "gap_params": {
    "default_sigma": [0.0025, 0.0150, 0.125, 0.125],
    "sparse_jitter": 1e-08,
    "do_copy_at_file": false,
    "sparse_separate_file": true,
    "energy_parameter_name": "DFT_energy",
    "force_parameter_name": "DFT_forces"
  }
}
`

**Comment by bernstei** [comment]
I agree that the parameters are different enough that it's worth having an example.  I'm also using turboSOAP with the multi-stage universal_SOAP heuristic GAP, so let me compare yours with mine, and confirm that everything looks right.

**Comment by Felixrccs** [comment]
[turbosoap_multistage.zip](https://github.com/libAtoms/workflow/files/10536316/turbosoap_multistage.zip)

Here is the full json file. that is probably easier to look at.
